tailwind.config.js

```diff
@@ -2,7 +2,11 @@
 module.exports = {
   content: ['./src/**/*.{js,jsx,ts,tsx}'],
   theme: {
-    extend: {},
+    extend: {
+      fontFamily: {
+        poppins: ['Poppins'],
+      },
+    },
   },
   plugins: [],
 };
```

Comment:
need importing for poppins font
@import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
:)))